app/models/comment.rb

```diff
@@ -179,6 +179,13 @@ def after_create
         end
       end
     end
+
+    return unless self.comment_owner&.is_tag_wrangler? && self.ultimate_parent.is_a?(Tag)
```

Comment:
Would it be at all possible to pull this out into a separate `after_create` callback? This existing one is... quite long and unhelpfully named, so it would be great if we could avoid making the problem worse by having `after_create :update_last_wrangling_activity` specifically for wrangling activity.